practice-02/src/index.html

```diff
@@ -17,18 +17,115 @@
         <section id="box-filter">
             <!-- Filter -->
             <div class="block-filter-main">
-                <div class="filter-total">
+                <!-- Filter Total -->
+                <div class="filter-item filter-total">
                     <div class="filter-item__title">Filter</div>
+                    <div class="filter-show show-total">
+                        <div class="show-total-main">
+                            <div class="show-total-item clearfix">
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item">
```

Comment:
If its not a link, Please dont make it's a link, using another HTML tag

Reply:
Update all similar cases

Reply:
I updated a link into button